projects/objects/trees/protos/BigSassafras.proto

```diff
@@ -34,12 +34,7 @@ PROTO BigSassafras [
             ]
           }
```

Comment:
The conversion of the points of `Coordinate` are correct, but I think you deleted the ` texCoord` node by error and it causes a bad mapping of the texture on the trunk